### DISCUSSION TITLE
A toolbar for the carousel

### DISCUSSION THREAD

**Discussion by arkjoseph** [fancyapps/ui#713] [Q&A]
Is it possible to insert the same toolbar for the gallery/fancybox on a carousel?  

Id like those toolbar buttons available regardless if a user enters fancybox.  If not, any work arounds or custom recommendations?  

Does this exist already with version 5? also, whats the roadmap toward v6?

**Comment by fancyapps** [comment]
Hi.

Sorry, this is not possible at the moment, but I can inform you that the development of v6 has made good progress and this feature is already possible in v6. It is difficult to set a roadmap or predict when it will be ready for release, but the current goal is to release the first beta early next year. You can follow the news on the Discord server.
